docs/data_management/lts/iam_and_policies.md

```diff
@@ -1,6 +1,86 @@
-# Sharing Buckets
+---
+toc_depth: 3
+---
 
-A major use for LTS is storage of data that should be accessible to multiple users from a lab or research group. By default, buckets are only visible and accessible to the owner of the bucket, and no mechanism exists to search for buckets other users have created.
+# LTS Identity and Access Management
+
+LTS Identity and Access Management (IAM) is a framework for managing identities, roles, and permissions in Long-Term Storage (LTS) solutions, ensuring secure and efficient access to storage spaces like [buckets](./index.md#terminology) or [objects](./index.md#terminology). Understanding access rights and permissions for LTS spaces is essential for effective data management and security.This section aims to clarify common misconceptions regarding ownership, steward roles, access control, and how bucket policies help manage permissions in LTS spaces.
```

Comment:
Needs a space after the full stop in "and security.This section".

Reply:
This has been fixed. Thank you.